### PR TITLE
Contextualmenu: Make item text overflow work as expected with flexbox.

### PR DESCRIPTION
**Pull request by joschect**
#### Pull request checklist

- [x] Include a change request file if publishing (Run `npmx change` to generate it.)
- [x] New feature, bugfix, or enhancement
- [x] Documentation update

#### Description of changes

Added in a flexbox so that contextualmenu now properly has flexbox.

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.scss [hunk 1]
```diff
@@ -94,17 +97,23 @@ a.ms-ContextualMenu-link {
...
 }
 
 .ms-ContextualMenu-iconColor {
   color: $ms-color-themePrimary;
 }
 
 .ms-ContextualMenu-itemText {
-  @include padding(0, 12px, 0, 4px);
+  margin: 0 4px;
+    vertical-align: middle;
```
Comment (mikewheaton) [inline packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.scss]: Minor indenting issues here.

File: packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.scss [hunk 2]
```diff
@@ -115,11 +124,11 @@ a.ms-ContextualMenu-link {
 }
 
 .ms-Icon.ms-ContextualMenu-submenuChevron {
```
Comment (mikewheaton) [inline packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.scss]: Is `.ms-ContextualMenu-submenuChevron` not specific enough here?

Comment (joschect) [inline packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.scss]: It appears to be specific enough. I changed it.

File: packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.scss (hunk 1 above)
Comment (joschect) [inline packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.scss]: Fixed

**Pull request merged**